tests/arp/conftest.py

```diff
@@ -242,17 +242,16 @@ def ip_and_intf_info(config_facts, intfs_for_test, ptfhost, ptfadapter):
         except ValueError:
             continue
 
-    # The VLAN interface on the DUT has an x.x.x.1 address assigned (or x::1 in the case of IPv6)
-    # But the network_address property returns an x.x.x.0 address (or x::0 for IPv6) so we increment by two to avoid conflict
+    # Increment address by 3 to offset it from the intf on which the address may be learned
     if intf_ipv4_addr is not None:
-        ptf_intf_ipv4_addr = increment_ipv4_addr(intf_ipv4_addr.network_address, incr=2)
+        ptf_intf_ipv4_addr = increment_ipv4_addr(intf_ipv4_addr.network_address, incr=3)
```

Comment:
do we see an issue with 2? if so, can you please give an example

Reply:
I didn't run the new test case with 2, since it would result in the neighbor solicitation for a target IP being sent on the interface the IP is learned on the ToR. For the scenario, we need the link the target IP is learned on to be standby, and the ToR should receive the NS on some other active link. The test will always pick the first interface on the ToR to send it, so I changed the IP address to one that would be learned on a different interface.

The other ARP test cases are still able to pass with this change.